multipage.js

```diff
@@ -116,7 +116,8 @@ for(var i=0; i<sections.length; i++) {
 
   // find the proper section
   section.node = header.parent();
-  while (section.node.get(0).tagName !== "section") {
+  //while (section.node.get(0).tagName !== "section") {
+  while (section.node.get(0) && section.node.get(0).tagName !== "section") {
```

Comment:
I would have done a hard undefined check if that's what your extra check is intending.